From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 16 Mar 2016 18:19:04 +0100
Subject: Revert "libata: Align ata_device's id on a cacheline"
Patch-mainline: never, kabi

This reverts commit 54709aebcea1709772a302a9592d52a8252bc08d, upstream
commit 4ee34ea3a12396f35b26d90a094c75db95080baa. It breaks kABI of a
structure that can be used widely from the external world. Adding some
space before devslp_timing may affect them all, unfortunately.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/linux/libata.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/libata.h b/include/linux/libata.h
index a445209be917..189c9ff97b29 100644
--- a/include/linux/libata.h
+++ b/include/linux/libata.h
@@ -712,7 +712,7 @@ struct ata_device {
 	union {
 		u16		id[ATA_ID_WORDS]; /* IDENTIFY xxx DEVICE data */
 		u32		gscr[SATA_PMP_GSCR_DWORDS]; /* PMP GSCR block */
-	} ____cacheline_aligned;
+	};
 
 	/* DEVSLP Timing Variables from Identify Device Data Log */
 	u8			devslp_timing[ATA_LOG_DEVSLP_SIZE];
-- 
2.7.3

